components/runners/ambient-runner/ambient_runner/bridges/claude/memory.py

```
@@ -0,0 +1,399 @@
+            "git",
+            *args,
+            cwd=cwd,
+            stdout=asyncio.subprocess.PIPE,
+            stderr=asyncio.subprocess.PIPE,
+        )
+        stdout, stderr = await proc.communicate()
+        if proc.returncode != 0:
+            error = stderr.decode().strip() or stdout.decode().strip()
+            return False, error
+        return True, stdout.decode().strip()
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)